userland/libtock/Makefile

```diff
@@ -9,7 +9,7 @@ AS := $(TOOLCHAIN)-as
 ASFLAGS += -mcpu=$(TOCK_ARCH) -mthumb
 
 CC := $(TOOLCHAIN)-gcc
-CFLAGS += -O3 -Wall -Werror -I$(CUR_DIR) -g -mcpu=$(TOCK_ARCH) -mthumb
+CFLAGS += -std=gnu1x -O3 -Wall -Werror -I$(CUR_DIR) -g -mcpu=$(TOCK_ARCH) -mthumb
```

Comment:
Change to `gnu11`. `gnu1x` is deprecated https://gcc.gnu.org/onlinedocs/gcc/C-Dialect-Options.html